### ISSUE TITLE
Faulty Core fixes: else control structures without braces

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#971]
#### Summary:
The autofixer for `else` ~~/ `elseif`~~ used without braces sometimes removes a new line which should be there.

#### Most likely cause:
Conflict between the `Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword` - _"Expected 1 space after ELSE keyword; newline found"_ - and the `Generic.ControlStructures.InlineControlStructure.NotAllowed` - _"Inline control structures are not allowed"_ - sniffs.

#### Examples:

File: various files

Example from `wp-login.php` around line 194:
```php
				if ( 'message' == $severity )
					$messages .= '	' . $error_message . "<br />\n";
				else
					$errors .= '	' . $error_message . "<br />\n";
```

Fixed as:
```php
				if ( 'message' == $severity ) {
					$messages .= '	' . $error_message . "<br />\n";
				} else {                  $errors .= '	' . $error_message . "<br />\n";
				}
```

Expected fix:
```php
				if ( 'message' == $severity ) {
					$messages .= '	' . $error_message . "<br />\n";
				} else {
					$errors .= '	' . $error_message . "<br />\n";
				}
```

Example from `wp-signup.php` around line 105:
```php
	if ( !is_subdomain_install() )
		echo '<label for="blogname">' . __('Site Name:') . '</label>';
	else
		echo '<label for="blogname">' . __('Site Domain:') . '</label>';
```

Fixed as:
```php
	if ( ! is_subdomain_install() ) {
		echo '<label for="blogname">' . __( 'Site Name:' ) . '</label>';
	} else {      echo '<label for="blogname">' . __( 'Site Domain:' ) . '</label>';
	}
```

Expected fix:
```php
	if ( ! is_subdomain_install() ) {
		echo '<label for="blogname">' . __( 'Site Name:' ) . '</label>';
	} else {
		echo '<label for="blogname">' . __('Site Domain:') . '</label>';
	}
```

**Comment by jrfnl** [comment]
Fix has been pulled upstream https://github.com/squizlabs/PHP_CodeSniffer/issues/1514.

**Comment by jrfnl** [comment]
Darn! Found yet another example which is related to this issue, but not (yet) fixed by the PR. I'll see if I can improve on this solution some more.

Example code:
```php
while ( isset($menu[$ptype_menu_position]) || in_array($ptype_menu_position, $core_menu_positions) )
	$ptype_menu_position++;
```

Is currently being fixed as:
Example code:
```php
while ( isset( $menu[ $ptype_menu_position ] ) || in_array( $ptype_menu_position, $core_menu_positions ) ) {        $ptype_menu_position++;
}
```

**Comment by jrfnl** [comment]
Ok, so I found the *real* cause of this issue with some help from upstream. It's not so much a bug in the upstream sniff, but a "feature" in our ruleset

PR #366 enabled the sniffing for control structure signatures, but explicitly **excludes** the rule to add a new line after the brace.

Removing that exclusion would fix this issue + #974.

@JDGrimes As you pulled that PR originally, how do you feel about removing the exclusion ?

Everyone else is of course also welcome to weight in.
